dpe/src/commands/certify_key.rs

```diff
@@ -151,7 +151,7 @@ impl CommandExecution for CertifyKeyCmd {
             derived_pubkey_y,
             cert_size,
             cert,
-            resp_hdr: ResponseHdr::new(DpeErrorCode::NoError),
+            resp_hdr: ResponseHdr::new(dpe.profile, DpeErrorCode::NoError),
```

Comment:
It might be nice to wrap this with a helper function in `DpeInstance`, in case a new call-site sets it incorrectly.

Reply:
That's a good idea, done.